src/At.vue

```diff
@@ -255,7 +255,12 @@ export default {
         } else {
           const { members, filterMatch, itemName } = this
           if (!keep && chunk.length>0) {
-            this.$emit('at', chunk)
+            const memberObj = Object.values(this.members).filter((i) => {
```

Comment:
I'm worried about cross-browser compatibility of `Object.values` (not supported in IE, etc)
https://caniuse.com/#search=object.values